### ISSUE TITLE
URLs should have hypens in their regex patterns and underscores in their names

### ISSUE THREAD

**Issue by atbaker** [TwilioDevEd/automated-survey-django#6]
[Mostly because Google says so](https://support.google.com/webmasters/answer/76329?hl=en), URLs should have hypens instead of underscores in their patterns.

Because the URL names in Django are Python strings, people usually use underscores for those.

So in this example: https://github.com/TwilioDevEd/automated-survey-django/blob/master/automated_survey/urls.py#L39-L41

It should look like this:

```
    url(r'^first-survey/',
        csrf_exempt(redirect_to_first_survey),
        name='first_survey'),
```

**Comment by jsantos17** [comment]
@atbaker thanks! I'll change. I somehow mixed them up.

**Comment by jsantos17** [comment]
Fixed by 2fbf7cb09b
